src/components/WinnerPage/Winner.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Winner.css';
+import trophy from './trophy2.0.png';
+import mewtwo from './150Mewtwo.png';
+import returnArrow from './return.png';
+
+
+
+function Winner(){
+    return(
+        <div className="winnerResult">
+            <div className = 'roll-in-blurred-left'><img className="trophy" alt ="#" src={trophy}/></div>
+                <div className = "v_1">
+                    <div className='slide-in-blurred-top'><div className = "winResultTitle"><h1>VICTORY</h1></div></div>
```

Comment:
Don't write text in uppercase, write it normally and use the css property `text-transform: uppercase;`